src/generate-note-controller.js

```diff
@@ -191,14 +192,14 @@ module.exports = function(scribe){
         //we need to figure out if our caret or selection is within a conflicting note
         var isWithinConflictingNote = false;
         config.get('selectors').forEach((selector)=>{
-          if((selector.tagName !== tagName) && isSelectionWithinNote(markers, selector.tagName)){
-            isWithinConflictingNote = true;
-          }
+        if((selector.tagName !== tagName) && isSelectionWithinNote(markers, selector.tagName)){
+        isWithinConflictingNote = true;
+        }
         });
 
         //if we ARE within a confilicting note type bail out.
         if(isWithinConflictingNote){
-          return;
+        return;
         }
         */
 
```

Comment:
Can you just remove the code rather than commenting it out? We can use git history if we need it again.